src/odemis/driver/lakeshore.py

```diff
@@ -451,6 +451,11 @@ def __init__(self, timeout=1):
         self._setpoint = 150  # K
         self._temperature = 100  # K
         self._heating = 0  # enum int 0,1,2, or 3
+        self._temperature_increase = False
+        
+        if os.path.exists("temp_increase.txt"):
```

Comment:
Don't put it in the `__init__()`, but directly at the `if`. So the user can add/remove the file at any time and the driver follows the requested behaviour.

For the path, currently it checks in the "current directory", which is not super obvious (it depends how you start the back-end). So I'd suggest using `os.path.join(model.BASE_DIRECTORY, ...)`, as we did for the simcam.